### PR TITLE
ACMS-796: Add new text format Full HTML

### PR DESCRIPTION
**Pull request by chandan-singh7929**
**Motivation**

Fixes #ACMS-796(https://backlog.acquia.com/browse/ACMS-796)

**Proposed changes**

We are creating new text format in acquia cms as its required by GDPR module

**Alternatives considered**

**Testing steps**

Install ACMS and make sure new text format is available and can be used by user.

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Review by vishalkhode1 — CHANGES_REQUESTED** [review]
@chandan-singh7929 We need to write update hook for the same in acquia_cms_common module, so that anyone already using Acquia CMS should get this configs updated.
cc: @kdruckman

**Comment by chandan-singh7929** [comment]
> @chandan-singh7929 We need to write update hook for the same in acquia_cms_common module, so that anyone already using Acquia CMS should get this configs updated.
> cc: @kdruckman

This is done.

File: modules/acquia_cms_common/acquia_cms_common.install [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Add new text format 'Full HTML'.
+ */
+function acquia_cms_common_update_8001() {
+  // Do not override existing full_html, if already provide.
+  $config_full_html = \Drupal::config('filter.format.full_html')->get('name');
+  ;
```
Comment (vishalkhode1) [inline modules/acquia_cms_common/acquia_cms_common.install:16]: @chandan-singh7929 Can you remove this extra line ?

Comment (chandan-singh7929) [reply modules/acquia_cms_common/acquia_cms_common.install]: Done.

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**